subsys/settings/Kconfig

```diff
@@ -104,3 +110,25 @@ config SETTINGS_FS_MAX_LINES
 	depends on SETTINGS && SETTINGS_FS
 	help
 	  Limit how many items stored in a file before compressing
+
+config SETTINGS_NVS_SECTOR_SIZE
+	int "Sector size of the NVS settings area"
+	default 4096
```

Comment:
I would default this to DT_FLASH_ERASE_BLOCK_SIZE, if possible

Reply:
It should be possible by using `default $(dt_hex_val, DT_FLASH_ERASE_BLOCK_SIZE)`

Reply:
Nice, I just learned something new and very interesting.